packages/suite/src/components/backup/BackupSeedCard.tsx

```diff
@@ -1,30 +1,45 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Icon, IconProps, variables, useTheme } from '@trezor/components';
+import { darken } from 'polished';
+import { Icon, IconProps, variables, useTheme, Checkbox } from '@trezor/components';
 
-const Card = styled.div<{ checked: boolean }>`
-    display: flex;
-    padding: 12px 24px;
+const StyledCheckbox = styled(Checkbox)`
+    position: absolute;
+    right: 0;
+    top: 20px;
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        top: auto;
+    }
+`;
 
+const Card = styled.div<{ checked: boolean }>`
+    position: relative;
+    padding: 24px;
     border-radius: 10px;
-    border: solid 1px ${props => (props.checked ? props.theme.TYPE_GREEN : props.theme.STROKE_GREY)};
+    border: solid 1.5px ${({ theme, checked }) => (checked ? theme.TYPE_GREEN : theme.STROKE_GREY)};
+    transition: all 0.2s ease-in-out;
```

Comment:
nit: specifically listing the properties is less error-prone than `all`. I think `transition: all` can cause inconsistent  colour transition when changing theme.

Reply:
Yes, I usually correct `all` myself, missed it this time :)

Reply:
302212e9f